apps/dashboard/src/components/creat-group-modal.tsx

```diff
@@ -145,40 +134,46 @@ export default function CreatGroupModal({
                 <ModalCloseButton />
                 <ModalBody>
                     {_step === 0 ? (
-                        <Flex
-                            h="300px"
-                            flexDir="column"
-                            justifyContent="space-around"
-                        >
-                            <FormControl>
-                                <FormLabel>Name</FormLabel>
-                                <Input
-                                    value={_groupName}
-                                    onChange={(e) =>
-                                        setGroupName(e.target.value)
-                                    }
-                                    placeholder="Give your group a title"
-                                />
-                            </FormControl>
-                            <FormControl>
-                                <FormLabel>Description</FormLabel>
-                                <Input
-                                    value={_groupDescription}
-                                    onChange={(e) =>
-                                        setGroupDescription(e.target.value)
-                                    }
-                                    placeholder="Enter details that will help you differentiate this group"
-                                />
-                            </FormControl>
-                            <Button
-                                onClick={submitGroupInfo}
-                                fontSize="lg"
-                                variant="solid"
-                                colorScheme="primary"
+                        <form onSubmit={nextStep}>
```

Comment:
This appear as a big diff, but the main change is adding `<form>` and indenting the existing by an extra tab 😃

Button type is changed to `submit` and validation are added to `input` controls directly